mobilesdk/zbox/storage.go

```diff
@@ -557,15 +564,48 @@ func CreateDir(allocationID, dirName string) error {
 
 // RevokeShare revoke authTicket
 //
-//  ## Inputs
-//  - allocationID
-//  - path
-//  - refereeClientID
+//	## Inputs
+//	- allocationID
+//	- path
+//	- refereeClientID
 func RevokeShare(allocationID, path, refereeClientID string) error {
-  a, err := getAllocation(allocationID)
-  if err != nil {
-    return err
-  }
-  return a.RevokeShare(path, refereeClientID)
+	a, err := getAllocation(allocationID)
+	if err != nil {
+		return err
+	}
+	return a.RevokeShare(path, refereeClientID)
 }
 
+// GetRemoteFileMap returns the remote
+//
+//	## Inputs
+//	- allocationID
+func GetRemoteFileMap(allocationID string) (string, error) {
+	a, err := getAllocation(allocationID)
+	if err != nil {
+		return "", err
+	}
+
+	ref, err := a.GetRemoteFileMap(nil, "/")
+	if err != nil {
+		return "", err
+	}
+
+	fileResps := make([]*fileResp, 0)
```

Comment:
Let's define the slice cap as `len(ref)` instead of `0`

Reply:
fixed in de0d888